### PR TITLE
Feature/pelagos 4587 create server api in node js

### PR DESCRIPTION
**Pull request by fightingtexasaggie**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,32 +1,51 @@
 const express = require('express');
-const app = express();
+const https = require('https');
+const fs = require('fs');
+const { Http2ServerRequest } = require('http2');
```
Comment (mickel1138) [inline app.js]: This is not being used anywhere?

File: app.js [hunk 2]
```diff
@@ -1,32 +1,51 @@
...
+
 const port = 3000;
+const sslMode = false;
+const baseDir = '';
 
-const fs = require('fs');
+const privateKeyFile = '';
+const sslCertFile = '';
```
Comment (mickel1138) [inline app.js]: If you want to paramaterize (not a word) these variables, I recommend using `yargs`
If not, you can at least delete privatekeyfile and sslcerfile as they are being re-defined later.

File: app.js [hunk 3]
```diff
@@ -1,32 +1,51 @@
...
+  const sslCertificate = fs.readFileSync(sslCertFile);
+
+  https.createServer({
+    key: privateKey,
+    cert: sslCertificate
+  }, app).listen(port);
+  // eslint-disable-next-line no-console
+  console.log(`Folder getter app listening at https://localhost:${port}`);
+} else {
+  app.listen(port, () => {
```
Comment (mickel1138) [inline app.js:46]: To use .env, do `yarn add dotenv` then add this line: `require('dotenv').config()` and use it.

File: app.js (hunk 1 above)
Comment (mickel1138) [inline app.js]: Make sure used packages are also installed and referenced in packages.json. (Even if they already exist in node_moduls)

File: app.js (hunk 2 above)
Comment (fightingtexasaggie) [reply app.js]: They're not being re-defined. privateKeyFile vs privateKey, sslCertFile vs sslCertificate, but I will move the files to arguments or env parameters of some sort.

**Review by praneethpr — APPROVED** [review]

**Review by mickel1138 — APPROVED** [review]

**Pull request merged**